org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/semanticTokens/SemanticHighlightReconcilerStrategy.java

```diff
@@ -82,6 +84,8 @@
 public class SemanticHighlightReconcilerStrategy
 		implements IReconcilingStrategy, IReconcilingStrategyExtension, ITextPresentationListener {
 
+    private static final int FULL_RECONCIL_TIMEOUT_S = 10;
```

Comment:
should this not be `FULL_RECONCILE_TIMEOUT_S`?

Reply:
ACK